pkgs/top-level/all-packages.nix

```diff
@@ -10836,6 +10836,7 @@ let
   firefox = callPackage ../applications/networking/browsers/firefox {
     inherit (gnome) libIDL;
     inherit (pythonPackages) pysqlite;
+    enableGTK3 = false;
```

Comment:
@edolstra I believe GTK3 should be enabled by default once everything is polished, so this line is a reminder.